layouts/shortcodes/version-selector.html

```diff
@@ -0,0 +1 @@
+<ul id="version-selector"></ul>
```

Comment:
`version-selector` is very generic. Would it be better to call this shortcode `content-version-selector` instead?